### ISSUE TITLE
Exception thrown for projects with no information under SDK 3.1

### ISSUE THREAD

**Issue by karolswdev** [CycloneDX/cyclonedx-dotnet#74]
Dear cyclonedx-dotnet team,

I discovered an issue with this product. 

When cyclonedx-dotnet attempts to retreive info for a local project that is listed as a dependency for another project but contains very little info about itself (such as the below) it blows up.

```xml
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <TargetFramework>netstandard2.1</TargetFramework>
    </PropertyGroup>

</Project>
```

It looks like it is a bug in the framework you use for reading NuSpec details.

I created a sample repo for your convenience to easily reproduce this issue:

https://github.com/karolswdev/cyclonedx-0-9-1-bug

I checked other issues but they were closed and opened quite a while back.

**Comment by karolswdev** [comment]
I have a hunch that this re-surfaced after multi-targeting .NET SDK 2.1 and 3.0. They must have changed the BaseNuSpecReader (can't guarantee the naming right now, when I debugged I basically had to decompile the framework to see where it possibly blew up) implementation. 

I can't look into this this week, but I'd suggest adding a unit test to capture this going forward.

**Comment by coderpatros** [comment]
Thanks for the repro @karolswdev I'll take a look.

**Comment by stevespringett** [comment]
Fixed in v0.9.2
